packages/ncids-css/packages/nci-card-group/internal/_nci-card-group.scss

```diff
@@ -76,3 +76,44 @@
 		}
 	}
 }
+
+.imageless-card-group {
```

Comment:
https://github.com/NCIOCPL/ncids/wiki/The-Way

Reply:
(My bad, I figured prototyping implied it wasn't getting merged or something to that effect and it could be kinda WIPish)

Reply:
just wanna make sure the copy pasta will be correct when the time comes